site/content/entry/citra-progress-report-2018-q2.md

```diff
@@ -0,0 +1,305 @@
+
+The majority of support requests we get are mostly due to users accidentally
+enabling or disabling some settings. But now with this implementation, we have
+default values for all our settings. This ensures that in case of any new
+settings changes, majority of the users (who haven't customized those settings)
+will now have the new default value automatically.
+
+## [Add support for encrypted games](https://github.com/citra-emu/citra/pull/4020) by [wwylele](https://github.com/wwylele)
+
+You can now directly use your encrypted game dumps in Citra, without having to
+decrypt them. This requires that the user provide their keys via a text file
+in `sysdata/aes_keys.txt`.
+
+Currently all known encryption methods are implemented except for the Seed
+crypto. Seed crypto is used in newer eshop games, which essentially uses an
+additional title-unique key that is stored outside of the ROM.
+
+## [Scripting support](https://github.com/citra-emu/citra/pull/4016) by [EverOddish](https://github.com/EverOddish)
+
+Scripting is an extremely useful feature which allows us to do things like
+debugging emulator internals, reverse engineering 3DS game internals,
+Tool-Assisted-Speedrunning (TAS), and cheat in games.
+(**Does not add actual Gameshark / AR Code cheat support**)
+
+Users can make their own scripts or use publicly available scripts to gain
+power over things like game screen display, inputs, and memory. Scripts can
+be used to modify the memory of games to get cheat-like behavior.
+
+Scripts can be used to read or modify the internal game state. EverOddish
+wrote a script that can read internal stats of the Pok&eacute;mon and
+automatically update image files for your current party. This feature is very
+useful for streamers who might want to display game information on their
+streams.
+
+## Movie - game input recording and playback ([#2882](https://github.com/citra-emu/citra/pull/2882), [#3922](https://github.com/citra-emu/citra/pull/3922)) by [danzel](https://github.com/danzel) and [zhaowenlan1779](https://github.com/zhaowenlan1779)
+
+This feature, not to be confused with video/audio recording, implements a way
+to record game inputs and plays them back. Every time the hardware reads an
+input device (buttons, touch screen, accelerometer, gyroscope, c-stick) we
+copy (when recording) or replace (when playing) the values. Once playback
+has finished, we release control of inputs so that the user can take control.
+
+{{< figure src="/images/entry/citra-progress-report-2018-q2/movie.png"
+    title="UI options to record" >}}
+
+For this, danzel heavily took inspiration from dolphin's system and zhaowenlan1779
+wrote the UI for this feature. This method results in `.ctm` (citra TAS movie)
+files which are playble in Citra. To play a game's `.ctm` file you would need
+to have the game and also set system time to the same starting value as in
```

Comment:
Is this time setting done automatically? Or is this a requirement on the user? This could be elaborated more.